Content.Shared/Humanoid/Prototypes/SpeciesPrototype.cs

```diff
@@ -108,13 +108,13 @@ public sealed partial class SpeciesPrototype : IPrototype
     ///     Characters younger than this appear young.
     /// </summary>
     [DataField]
-    public int YoungAge = 30;
+    public int YoungAge = 40;
```

Comment:
Add `// DeltaV` comments explaining the change
Also would it not be better to override these on the individual species and give them appropriate values? (It's fine to leave this as is, just asking)